CONTRIBUTING.md

```diff
@@ -0,0 +1,4 @@
+Contributing to Unikraft
+=======================
+
+Please refer to the `CONTRIBUTING.md` file in the main Unikraft repository.
```

Comment:
Also use md format here instead of rst, and send people to the contributing section on the website (https://unikraft.org/docs/contributing/)